### ISSUE TITLE
Feature request: Pipeline sharing - anyone with URL

### ISSUE THREAD

**Issue by ewels** [seqeralabs/nf-tower#265]
Maybe you guys are already planning this, but it would be fab to have a check box / option to share a workflow run with _anyone_ as long as they have the URL and are logged in. Then if checked the workflow would essentially be shared with them (as in, appear in the wf list) once they have viewed it.

I'm thinking a bit like how you can share a url with anyone from google drive / dropbox etc.

**Comment by pditommaso** [comment]
Yes, we had some thoughts about this, likely it will be included along with the teams & orgs feature.
